Templates/BaseGame/game/tools/projectImporter/scripts/projectImporter.tscript

```diff
@@ -556,23 +556,49 @@ function processLegacyField(%line, %originalFieldName, %newFieldName)
       %targetFilename = sanitizeFilename(%value);
       
       if(isObject(%targetFilename))
+         {
```

Comment:
These misaligned indents starting here and onward make my brain hurt...

Reply:
Yeah, no positive what happened there, will get it corrected